modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java

```diff
@@ -319,10 +320,26 @@ protected Map<String, String> getTokens() throws Exception {
 	}
 
 	protected void initRegexTransformerListener() {
-		TransformerListener transformerListener =
-			_journalTransformerListenerRegistry.getTransformerListener(
-				"com.liferay.journal.internal.transformer." +
-					"RegexTransformerListener");
+		List<TransformerListener> transformerListeners =
```

Comment:
@JanisZhang Please mimic the solution at https://github.com/liferay-core-infra/liferay-portal/blob/8567ab84e3ba352f600659394a4dee861a74cdaf/modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java#L6978